src/script/page/MainContent/panels/preferences/accountPreferences/FileInput.tsx

```diff
@@ -20,30 +20,32 @@
 import React from 'react';
 import {TIME_IN_MILLIS} from 'Util/TimeUtil';
 
-interface FileInputProps extends React.HTMLProps<HTMLInputElement> {
+interface FileInputProps {
   fileTypes?: string[];
   onFileChange?: (files: FileList) => void;
 }
 
-const FileInput: React.FC<FileInputProps> = React.forwardRef(({fileTypes = ['*'], onFileChange, ...rest}, ref) => {
-  return (
-    <input
-      ref={ref}
-      type="file"
-      accept={fileTypes.join(',')}
-      onChange={({target}) => {
-        if (target.files.length > 0) {
-          onFileChange?.(target.files);
-          window.setTimeout(() => {
-            target.value = null;
-          }, TIME_IN_MILLIS.SECOND);
-        }
-      }}
-      onFocus={({target}) => target.blur()}
-      {...rest}
-    />
-  );
-});
+const FileInput = React.forwardRef<HTMLInputElement>(
+  ({fileTypes = ['*'], onFileChange, ...rest}: FileInputProps, ref) => {
+    return (
+      <input
+        ref={ref}
+        type="file"
+        accept={fileTypes.join(',')}
+        onChange={({target: {files, value}}) => {
+          if (files && files.length > 0) {
```

Comment:
```suggestion
          if (files?.length > 0) {
```

Reply:
Can't shorten it this way, `files?.length` might be `undefined`